client/my-sites/email/email-different-domain-owner-message/index.tsx

```diff
@@ -0,0 +1,53 @@
+import { useTranslate } from 'i18n-calypso';
+import TrackComponentView from 'calypso/lib/analytics/track-component-view';
+import { recordTracksEvent } from 'calypso/lib/analytics/tracks';
+
+type EmailDifferentDomainOwnerMessageProps = {
+	source: string;
+};
+
+export const EmailDifferentDomainOwnerMessage = (
+	props: EmailDifferentDomainOwnerMessageProps
+) => {
+	const { source } = props;
+
+	const translate = useTranslate();
+
+	const onClickLink = ( eventType: 'login' | 'support' ) => {
+		const properties = {
+			action: eventType,
+			source,
```

Comment:
@AllTerrainDeveloper Let's also send the property `context: 'different-owners'`, we can set the context for the other non-owner and non-domain-owner components that you built as well.